src/blas/impl/KokkosBlas_trtri_spec.hpp

```diff
@@ -69,7 +69,7 @@ struct trtri_eti_spec_avail {
                                         MEM_SPACE)                           \
   template <>                                                                \
   struct trtri_eti_spec_avail<                                               \
-      Kokkos::View<int, LAYOUTA, Kokkos::HostSpace,                          \
+      Kokkos::View<int, Kokkos::LayoutRight, Kokkos::HostSpace,              \
```

Comment:
@lucbv: I think this removes LayoutLeft ETIs; was that intended?

Reply:
There should be no difference between layout left and layout right since this is a rank 0 view, i.e there only one entry. At this point we are relying on Kokkos Core to provide appropriate compatibilities.